src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -305,6 +305,18 @@ private bool ReferencesWinMD(MetadataReader mdReader)
             return false;
         }
 
+        private bool HasILCode(PEReader peReader, MetadataReader mdReader)
+        {
+            foreach (var methoddefHandle in mdReader.MethodDefinitions)
+            {
+                MethodDefinition methodDef = mdReader.GetMethodDefinition(methoddefHandle);
+                if (methodDef.RelativeVirtualAddress > 0)
```

Comment:
(nit: we put braces around all blocks. I mentioned this in the original PR, but it got lost in the mix of tons of more substantive feedback)

Reply:
Added